cli/src/commands/util.rs

```diff
@@ -36,7 +36,7 @@ pub(crate) enum UtilCommand {
 /// Print a command-line-completion script
 #[derive(clap::Args, Clone, Debug)]
 pub(crate) struct UtilCompletionArgs {
-    /// Print a completion script for Bash
+    /// Print a completion script for Bash (default)
```

Comment:
Btw, we should also also make these options mutually exclusive; `jj util completion --fish --zsh` doesn't seem very useful :) That makes me wonder if we should instead make it `jj util completion --shell=fish`. It's a bit more to type, so it might not be worth it. Anyway, that is out of scope for this PR.

Reply:
i think `-s fish` isn't much longer, and i think it's rare for people to type this out anyway, i'd expect people to set it up once and put it in ~/.bashrc or whatever. i'll make a PR for that :)

Reply:
Why not just `jj util completion fish`? It could even be a subcommand.

I'd also keep supporting `--fish` for a while (as a hidden option?). Maybe print a warning at some point. It's a painful thing to change for docs, the string is baked in somewhere in people's configs they forgot about.

Obviously, they should be mutually exclusive.

Reply:
Oh, that sounds like a good idea. What do you think, jyn?

Reply:
oh i like that! that matches what `zoxide` does as well :) and it lets us extend the commands with shell-specific options in the future if we need to do that for some reason.
:+1: to keeping the existing options as hidden for a while